SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+# google policy
+# microsoft security
```

Comment:
Three titles for the page don't make sense. Perhaps they're a sign that this PR is spam... though the rest of it seems innocuous enough...